src/idlcxx/src/backend.c

```diff
@@ -270,12 +270,9 @@ idl_walk_tree(
       matches_target_file = true;
     }
 
-    if (matches_target_file)
+    if (matches_target_file && (target_node->mask & mask))
```

Comment:
Is it okay if only one of the bytes of mask matches, or should all of them match?

Reply:
@reicheratwork The mask in here is meant to be a mask, not an exact match. So I could request to walk over the parse tree and match for any of the following types: (IDL_MODULE | IDL_STRUCT | IDL_UNION). These bits are fully mutually exclusive, but I don't want to make three separate walks to visit the three separate types.

Reply:
Check, this was more of a sanity check from my end, whether this was intentional or whether you might have missed something.
In this case, I would say this comment can be considered to be answered to satisfaction.